xrootd/resources/test-scitokens-origin-read.cfg

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2024, Pelican Project, Morgridge Institute for Research
```

Comment:
Since we're officially changing the scitokens config template, can you update the copyright statements?